server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQFixture.java

```diff
@@ -0,0 +1,49 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.queue.rabbitmq;
+
+import static com.jayway.awaitility.Duration.FIVE_HUNDRED_MILLISECONDS;
+import static com.jayway.awaitility.Duration.ONE_MINUTE;
+
+import com.jayway.awaitility.Awaitility;
+import com.jayway.awaitility.Duration;
+import com.jayway.awaitility.core.ConditionFactory;
+import com.rabbitmq.client.AMQP;
+
+public class RabbitMQFixture {
+    public static final boolean DURABLE = true;
+    public static final boolean AUTO_ACK = true;
+    public static final AMQP.BasicProperties NO_PROPERTIES = null;
+    public static final String EXCHANGE_NAME = "exchangeName";
+    public static final String ROUTING_KEY = "routingKey";
+    public static final String DIRECT = "direct";
+    public static final boolean EXCLUSIVE = true;
+    public static final boolean AUTO_DELETE = true;
+    public static final String WORK_QUEUE = "workQueue";
+
+    public static Duration slowPacedPollInterval = FIVE_HUNDRED_MILLISECONDS;
```

Comment:
should we name static field in upper case?

Reply:
In our practive, CalmyAwait and related stuff is the exeption to that rule. (Can't explain why).

Reply:
I would argue this exception should be removed. @trantienduchn : would you mind doing another PR to change that ? And BTW, I'm surprised checkstyle doesn't catch that.